### PR TITLE
Added bxt_novis

### PR DESCRIPTION
**Pull request by chinese-soup**
Same principle as in og BunnymodXT.

**bxt_novis 0:**
c1a00010
**bxt_novis 1:**
c1a00009

### CODE REVIEW

File: src/hooks/engine.rs [hunk 1]
```diff
@@ -204,6 +210,11 @@ pub static LoadEntityDLLs: Pointer<unsafe extern "C" fn(*const c_char)> = Pointe
     ]),
     my_LoadEntityDLLs as _,
 );
+
+pub static Mod_LeafPVS: Pointer<
+    unsafe extern "C" fn(leaf: *mut mleaf_s, model: *mut model_s) -> *mut c_void,
+> = Pointer::empty(b"Mod_LeafPVS\0");
+
```
Comment (YaLTeR) [inline src/hooks/engine.rs:217]:
```suggestion
pub static Mod_LeafPVS: Pointer<
    unsafe extern "C" fn(*mut mleaf_s, *mut model_s) -> *mut c_void,
> = Pointer::empty(b"Mod_LeafPVS\0");
```

File: src/modules/mod.rs [hunk 2]
```diff
@@ -43,6 +44,7 @@ pub static MODULES: &[&dyn Module] = &[
     &cvars::CVars,
     &demo_playback::DemoPlayback,
     &fade_remove::FadeRemove,
+    &novis::NoVis,
     &force_fov::ForceFov,
     &hud_scale::HudScale,
     &module_list::ModuleList,
```
Comment (YaLTeR) [inline src/modules/mod.rs:50]:
```suggestion
    &force_fov::ForceFov,
    &hud_scale::HudScale,
    &module_list::ModuleList,
    &novis::NoVis,
```

**Review by YaLTeR — COMMENTED** [review]
This and run cargo fmt again just in case, then it's good.

**Comment by chinese-soup** [comment]
Done.

File: src/hooks/engine.rs [hunk 3]
```diff
@@ -204,6 +210,10 @@ pub static LoadEntityDLLs: Pointer<unsafe extern "C" fn(*const c_char)> = Pointe
     ]),
     my_LoadEntityDLLs as _,
 );
+
+pub static Mod_LeafPVS: Pointer<unsafe extern "C" fn(*mut mleaf_s, *mut model_s) -> *mut c_void> =
+    Pointer::empty(b"Mod_LeafPVS\0");
+
```
Comment (YaLTeR) [inline src/hooks/engine.rs:216]:
```suggestion
pub static Mod_LeafPVS: Pointer<unsafe extern "C" fn(*mut mleaf_s, *mut model_s) -> *mut c_void> =
    Pointer::empty(b"Mod_LeafPVS\0");
```
I also removed the blank lines in my suggestion (although github seems to eat the last one for some reason)

Comment (chinese-soup) [reply src/hooks/engine.rs]: github'd

**Comment by YaLTeR** [comment]
Thanks!

**Pull request merged**